dashboard/lib/services/script_seed.rb

```diff
@@ -175,22 +186,63 @@ def self.import_lessons(lessons_data, seed_context)
     Lesson.where(script: seed_context.script)
   end
 
+  def self.import_lesson_activities(activities_data, seed_context)
+    activities_to_import = activities_data.map do |activity_data|
+      lesson_id = seed_context.lessons.select {|l| l.key == activity_data['seeding_key']['lesson.key']}.first&.id
```

Comment:
Should there be a check that this lesson actually exists? Or is that what `seed_context` is for?

Reply:
yep, that's what seed_context is for. it prevents us from having to go to the DB to see if the lesson exists.

Reply:
Make sense -- thanks for the explanation!